docs/guides/mongodb/dr/overview/index.md

```
@@ -0,0 +1,329 @@
+primary in a surviving DC automatically, and the single connection endpoint follows
+the new writable DC.
+
+This page is the conceptual overview and a quick start. See also:
+
+- [DC-DR User Guide](/docs/guides/mongodb/dr/guide/index.md) for every aspect of
+  running in DC-DR mode (components, status, connecting, monitoring, switchover,
+  failback, day-2 ops).
+- [DC-DR Runbook](/docs/guides/mongodb/dr/runbook/index.md) for what to do in each
+  operational scenario.
+
+> **New to KubeDB?** Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)